packages/deploy-helpers/deploy-dao.js

```diff
@@ -0,0 +1,42 @@
+const Kernel = artifacts.require('@aragon/os/build/contracts/kernel/Kernel')
```

Comment:
This logic almost all Aragon apps use it. How do you think it would be the best way to expose it?